### PR TITLE
use sphinx-design tabs

### PR DESCRIPTION
**Pull request by GrahamSH-LLK**
incomplete, I still have 15 files to go
however the first portion is done and I'm heading to bed

### CODE REVIEW

**Review by sciencewhiz — CHANGES_REQUESTED** [review]

File: source/docs/contributing/frc-docs/style-guide.rst [hunk 1]
```diff
@@ -105,25 +105,23 @@ RLI (Remote Literal Include)
 
 When possible, instead of using code blocks, an RLI should be used.  This pulls code lines directly from GitHub, most commonly using the example programs.  This automatically keeps the code up to date with any changes that are made.  The format of an RLI is:
 
-.. code-block:: ReST
+.. tab-set-code::
```
[the remark this answers is not in the document]
Comment (GrahamSH-LLK) [reply source/docs/contributing/frc-docs/style-guide.rst:108]: yep i fixed that locally but haven't pushed yet. thanks!

**Comment by GrahamSH-LLK** [comment]
Should be done/ ready for review

**Comment by sciencewhiz** [comment]
It would be good to add something to the style guide about how to use tabs. It's not there now, but since the methods are changing I think it makes sense to document now

**Comment by Daltz333** [comment]
Do the `tab-set-code` actually sync across all tabs across a page?

**Comment by sciencewhiz** [comment]
I had tested a few and they did, but I haven't tried multiple browsers

**Comment by sciencewhiz** [comment]
sphinx-tabs can be removed as a dependency for poetry, right?

**Comment by GrahamSH-LLK** [comment]
> sphinx-tabs can be removed as a dependency for poetry, right?

yes

**Comment by GrahamSH-LLK** [comment]
I added to style guide and fixed the conflicts

**Review by Daltz333 — APPROVED** [review]

**Pull request merged**